api/src/main/java/com/ford/labs/retroquest/v2/columns/ColumnCombinerService.java

```diff
@@ -63,10 +62,7 @@ public ColumnCombinerResponse aggregateResponse(String teamId) {
         var actionItemColumnResponse = ColumnResponse.builder()
                 .title("Action Item")
                 .topic("action")
-                .items(ItemSorterResponse.builder()
-                        .completed(actionItems.stream().filter(ActionItem::isCompleted).collect(Collectors.toList()))
-                        .active(actionItems.stream().filter(a -> !a.isCompleted()).collect(Collectors.toList()))
-                        .build())
+                .items(new ArrayList<>(actionItems))
```

Comment:
I don't believe we need to wrap the `actionItems` in a new ArrayList. We should just pass them in directly.

Reply:
Doing anything else seems to break the tests

Reply:
Alright, we can merge this in as-is for now and take a look at it later. There's likely some other code elsewhere that's attempting to mutate the list (the immediate actionItems list is immutable) which it probably doesn't need to do.